### PR TITLE
remove schedule source

### PR DESCRIPTION
**Pull request by honnix**
@fabriziodemaria PTAL

### CODE REVIEW

File: styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java [hunk 1]
```diff
@@ -309,16 +305,7 @@ void awaitWorkflowInstanceCompletion(WorkflowInstance workflowInstance) {
   private void printTime() {
     LOG.info("The time is {}", now);
   }
-
-  private ScheduleSourceFactory fakeScheduleSource() {
-    return (changeListener, removeListener, environment, exec) ->
-        /* start */ () -> {
-      workflowChangeListener = changeListener;
-      workflows.forEach(changeListener);
```
Comment (bergman) [inline styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java]: these three fields can be deleted from the class too

Comment (honnix) [reply styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java]: Yeah. Good catch. I missed that.

**Comment by fabriziodemaria** [comment]
Code-wise it looks good to me, but we need to revisit the `README`, in particular the sections `Setup` and `Workflow schedule configuration`. Also, we need to remove the configuration entry `styx.source.local.dir` in the `styx-standalone` sub-module.

**Comment by honnix** [comment]
@fabriziodemaria will fix.

**Comment by honnix** [comment]
@fabriziodemaria more comments?

**Comment by honnix** [comment]
@fabriziodemaria ping

**Comment by fabriziodemaria** [comment]
No more comments on my side, looks good.

**Pull request merged**